docs/data-model/EA-Diagram.svg

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Comments:
- A user may have different types of work assignments
  - As a teacher, zero or more assignments that they created.
  - As a student, zero or more assignments that they are doing. In this case, probably the relation isn't with an assignment but instead to an "assignment resolution" or something similar.